images/cilium-test/install-deps.sh

```diff
@@ -16,6 +16,7 @@ ubuntu_packages=(
   gnupg
   grep
   jq
+  python-is-python3
```

Comment:
this package is symlinks /usr/bin/python to python3, so do we need to install python3 separately ?

Reply:
The `python-is-python3` package has `python3` [as dependency](https://packages.ubuntu.com/focal/python-is-python3), so no need to force mention it separately :D